src/main/java/com/zendesk/maxwell/schema/MysqlSchemaStore.java

```diff
@@ -86,6 +76,22 @@ private MysqlSavedSchema restoreOrCaptureSchema() throws SchemaStoreException {
 		}
 	}
 
+	public MysqlSavedSchema recaptureSchema() throws SQLException {
```

Comment:
this should just be named `captureSchema`

Reply:
There is `captureSchema()` defined in the base class `AbstractSchemaStore`. Any other suggestions?

Reply:
`captureAndSaveSchema()` describes what it does pretty accurately.  But also you could just override `captureSchema` and have `MysqlSchemaStore#captureSchema()` do the saving as well...

Reply:
`captureAndSaveSchema` sounds good!